#1085-remove email and text from footer

hasham-qaiser opened the pull request:
Removed EA email and text from footer component, the logo looks centered to the other links in the footer.

sydneywalcoff reviewed (changes requested):
Hey @hasham-qaiser ! Looking great so far!

<img width="728" alt="Screen Shot 2024-01-17 at 3 11 00 PM" src="https://github.com/hackforla/expunge-assist/assets/75542938/424e6635-83c2-41b1-955f-91f96f24cd2f">


The only thing I noticed is that between ~640px and 767px, the footer rearranges differently than design requested. Can you adjust? 

<img width="511" alt="Screen Shot 2024-01-17 at 3 12 22 PM" src="https://github.com/hackforla/expunge-assist/assets/75542938/fa08ad32-4b17-4abd-bc76-41bbf405e87e">


If you're not able to fit the content comfortably in the 900px breakpoint layout, have it follow the 600px layout instead.

hasham-qaiser commented:
hey @sydneywalcoff so i attempted to adjust the breakpoints so the rightContainer sits in a column at that screen size, i don't believe they will fit in a row at that size. The issue im seeing now is that if you go a littler smaller the links have a larger space/padding between them and footer div itself is bigger, not sure if thats a problem with the design team

sydneywalcoff commented:
@hasham-qaiser you can just have it use the 600px breakpoint design instead.

hasham-qaiser commented:
https://github.com/hackforla/expunge-assist/assets/117246204/e1abe3dc-3d2e-45c4-8a9a-fc24dc41fe9a

here is a screen recording:

hasham-qaiser commented:
@sydneywalcoff My last push, implements the 600px breakpoint into a column style. Let me know if it looks ok!

sydneywalcoff commented on products/statement-generator/src/components-layout/AppFooter.tsx:
```diff
@@ -45,6 +45,11 @@ const useStyles = makeStyles(({ breakpoints, palette, spacing }) =>
         marginLeft: 'auto',
       },
+      // Add custom media query for screen sizes between 640px and 767px
+      '@media (min-width: 640px) and (max-width: 767px)': {
```
this is great, but instead of using the literal media query, can you translate it to the use the material UI equivalent: `breakpoints.up/down()` [here are the docs](https://mui.com/material-ui/customization/breakpoints/)

sydneywalcoff commented on products/statement-generator/src/components-layout/AppFooter.tsx:
```diff
@@ -94,6 +99,9 @@ const useStyles = makeStyles(({ breakpoints, palette, spacing }) =>
       [breakpoints.down(breakpoints.values.sm)]: {
         display: 'none',
       },
+      '@media (min-width: 640px) and (max-width: 767px)': {
```
this too please

sydneywalcoff reviewed (changes requested):
@hasham-qaiser Looks good! If you could just refactor the media queries to use the Material UI equivalent. I left docs below.

sydneywalcoff commented on products/statement-generator/src/components-layout/AppFooter.tsx:
```diff
@@ -43,9 +43,18 @@ const useStyles = makeStyles(({ breakpoints, palette, spacing }) =>
+        flexDirection: 'row',
+      },
+
+      [breakpoints.between('xs', 'sm')]: {
+        flexDirection: 'column',
+      },
+
       [breakpoints.down(breakpoints.values.sm)]: {
         flexDirection: 'column',
       },
```
I think these are doing the same thing. Can we remove the one you added?

sydneywalcoff commented on products/statement-generator/src/components-layout/AppFooter.tsx:
```diff
@@ -91,6 +100,14 @@ const useStyles = makeStyles(({ breakpoints, palette, spacing }) =>
+        display: 'flex',
+      },
+
+      [breakpoints.between('xs', 'sm')]: {
+        display: 'none',
+      },
+
       [breakpoints.down(breakpoints.values.sm)]: {
         display: 'none',
       },
```
these also look to do the same thing.

hasham-qaiser replied:
these actually do not do the same thing. one is for the line between the links  the other is for the right column. `.between` is for screen sizes between two sizes, `.down` is for small and under.  if i remove .between for the line breakpoints the lines show up in a column like so: 
![Screenshot 2024-01-31 at 9 07 45 PM](https://github.com/hackforla/expunge-assist/assets/117246204/eb899a0d-62f2-4278-b6c3-ed2adf8b0937)

sydneywalcoff reviewed (changes requested):
@hasham-qaiser A couple notes please!

hasham-qaiser commented:
> @hasham-qaiser A couple notes please!

removed some lines i added previously, thought of a simple fix right before bed last night and just made another push. Hopefully this solves it
